WORKSPACE.bazel

```diff
@@ -38,9 +33,9 @@ node_repositories(
     "https://nodejs.org/dist/v{version}/{filename}",
   ],
   yarn_repositories = {
-    "1.21.1": ("yarn-v1.21.1.tar.gz", "yarn-v1.21.1", "d1d9f4a0f16f5ed484e814afeb98f39b82d4728c6c8beaafb5abc99c02db6674"),
+    "1.22.19": ("yarn-v1.22.19.tar.gz", "yarn-v1.22.19", "732620bac8b1690d507274f025f3c6cfdc3627a84d9642e38a07452cc00e0f2e"),
   },
-  yarn_version = "1.21.1",
```

Comment:
I think that should match what we have on our package.json. I'm not opposed to it but maybe we should update both sides?